src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## Unreleased
 
+* Updated `http.request.method` to match specification guidelines.
+  * For activity, request method will be set on an additional tag
```

Comment:
Place the if condition before for better readability.

For activity, if the method does not belong to one of the know values, the request method will be set on an additional tag...

Reply:
Same suggestion for metrics as well.

Reply:
Done